bitnami/fluentd/templates/forwarder-daemonset.yaml

```
@@ -111,8 +111,13 @@ spec:
           volumeMounts:
             - name: fluentd-config
               mountPath: /opt/bitnami/fluentd/conf
+            {{- if .Values.forwarder.persistence.enabled }}
+            - name: {{ include "fluentd.fullname" . }}-buffer
```

I'd use:
```suggestion
            - name: fluentd-buffer
```
I mean, I don't see any advantage on having this name based on the `fluentd.fullname` macro.